gradlew

```diff
@@ -28,7 +44,7 @@ APP_NAME="Gradle"
 APP_BASE_NAME=`basename "$0"`
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m" "-Xms64m"'
```

Comment:
I think 64bit is the default size, no ?

Reply:
I didn't change this — it's part of the new script installed by gradle wrapper.

Reply:
ah ok
and thanks for the commits btw !

Reply:
task fails now ...

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':HO:generatePlist'.
> No value has been specified for property 'MacAppBundlePlugin jreHome'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 1s
```

Reply:
Where do you see that failure?

Reply:
I disapeared when I upgraded macapp graddle plugin ... seems all good now